flapjack/ext.py

```diff
@@ -40,9 +41,9 @@ def flatpak(command, *args, output=False, code=False):
     if config.user_installation() and _takes_user_arg(command):
         user_arg = ['--user']
 
-    cmdline = (['flatpak', command] + user_arg + list(args))
+    cmdline = ['flatpak', command] + user_arg + list(args)
     if output:
-        return subprocess.check_output(cmdline)
+        return subprocess.check_output(cmdline, universal_newlines=True)
```

Comment:
The `universal_newlines` parameter is adding a `\n` before `http://` if using the current example config. This makes `flapjack setup` fail using the example INI file.

```
subprocess.CalledProcessError: Command '['flatpak', 'remote-add', '--if-not-exists', '--from', 'flapjack-source', '\nhttp://endlessm.github.io/eos-knowledge-lib/eos-sdk-nightly.flatpakrepo']' returned non-zero exit status 1
```

We should remove the newline after the equal sign here:
```
# sdk_repo_name = eos-sdk-nightly
sdk_repo_definition =
    http://endlessm.github.io/eos-knowledge-lib/eos-sdk-nightly.flatpakrepo
```

Reply:
Good catch. However I think it's better to strip whitespace from config keys, in order to avoid this problem in the future. I've added a commit for that.

Reply:
@ptomato good point. Also, I just noticed this is not an issue introduced by these changes, The universal_whitespace parameter is for output only. Testing your fix to strip whitespace. If that works I'll merge the PR.